### PR TITLE
[#2257] Add analytics demo

### PR DESCRIPTION
**Pull request by pascal-airy**

### CODE REVIEW

File: infrastructure/terraform/demos/analytics/variables.tf [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    default = "false"
+  
+}
+
+variable "tags" {
+  default = {}
+}
+
+variable "core_version" {
+  default = "0.31.0"
```
Comment (pascal-airy) [inline infrastructure/terraform/demos/analytics/variables.tf:36]: I still need a way to make this point to `latest`

File: docs/docs/guides/analytics-demo.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+browser:
+
+`kubectl get svc proxy-public`
+
+A login window should appear. Enter your name and simply click login.
+
+<img alt="jupyterhub"
+src={useBaseUrl('img/guides/analytics-demo/jupyterhub.png')} />
+
+### Creating a Pandas dataframe from the data stored on S3
```
Comment (ljupcovangelski) [inline docs/docs/guides/analytics-demo.md:135]: Perhaps a shorter title would look better in the content sidebar?

File: docs/docs/guides/analytics-demo.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+Open the `create_df_from_parquet.ipynb` and run the entire notebook. You now
+have a `df` object containing all the messages in the Airy
+`application_communication_messages` format.
+
+<img alt="fake_data_df"
+src={useBaseUrl('img/guides/analytics-demo/fake_data_df.png')} />
+
+### Loading the demo data into your platform
```
Comment (ljupcovangelski) [inline docs/docs/guides/analytics-demo.md:144]: Perhaps a shorter title would look better in the content sidebar?

File: infrastructure/terraform/demos/analytics/variables.tf (hunk 1 above)
Comment (ljupcovangelski) [reply infrastructure/terraform/demos/analytics/variables.tf:36]: Perhaps read the VERSION file (as the guide implies git clone) and render the content in a:
```
locals {
  version = data.template_file.version.rendered
}
```
Then you can use it as `local.version`.

An alternative I think would be to somehow use https://airy-core-binaries.s3.amazonaws.com/stable.txt

File: infrastructure/terraform/demos/analytics/outputs.tf [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+output "copy_config_dns" {
+  description = "Command for copying the kubeconfig for connecting using DNS address"
+  value       = "To copy the kubectl config file using DNS record, run: 'scp ${module.minikube.ssh_user}@${module.minikube.dns}:${module.minikube.kubeconfig_dns} .kubeconfig'"
+}
+
+output "copy_config_ip" {
+  description = "Command for copying the kubeconfig for connecting using IP address"
+  value       = "To copy the kubectl config file using IP address, run: 'scp ${module.minikube.ssh_user}@${module.minikube.public_ip}:${module.minikube.kubeconfig_ip} .kubeconfig'"
+} */
```
Comment (ljupcovangelski) [inline infrastructure/terraform/demos/analytics/outputs.tf:19]: This should be commented out perhaps.

**Review by ljupcovangelski — DISMISSED** [review]
Great work!

File: docs/docs/guides/analytics-demo.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+dashboard, giving you an overview of the demo data. To drill down
+even further, a Jupyter Notebook is deployed for you to run your first
+conversational analytics.
+
+</TLDR>
+
+## Demo Dashboard
+
+The following
+[Metabase Demo Dashboard](https://dashboard.airy.co/dashboard/14?date_range=2021-08-24~2021-08-24)
```
[the remark this answers is not in the document]
Comment (pascal-airy) [reply docs/docs/guides/analytics-demo.md:22]: oh thats cool

**Review by ljupcovangelski — APPROVED** [review]

**Pull request merged**